pipelines/tasks/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '2'
+services:
```

Comment:
It seems kind of unnecessary in this case -- you can limit the services started by using `servicename` in the compose call which seems to be what we'd want to do here. At least for me it makes things more confusing when trying to figure out "where can I see how this is invoked".

We had a similar set up on other projects when we used celery. The verdict seems still out on the best practices here definitely, I just think this is more confusing than separating out docker-compose files, especially when we'll want to be invoking jobs from the server code so we will need both sets of containers up at some point. At a minimum, moving the docker-compose file to the top-level would be better I think.

The best use-case we've had for using extends in the past is when we need to override something (environment, container, etc.) to run tests.